src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -197,6 +198,7 @@ export class RelayFeeCalculator {
   async gasFeePercent(
     deposit: Deposit,
     amountToRelay: BigNumberish,
+    amountToSimulate: BigNumberish,
```

Comment:
Updated to make this a boolean that defaults to true. I agree that we may want to add an override type to these functions, but it may be better suited in a separate PR.